xmake/modules/package/manager/pacman/find_package.lua

```diff
@@ -65,28 +142,54 @@ function main(name, opt)
             has_includes = true
         end
     end
-
-    -- get pkgconfig file
-    local pkgconfig_file = pkgconfig_files[name]
-    if not pkgconfig_file then
-        for _, file in pairs(pkgconfig_files) do
-            pkgconfig_file = file
-            break
-        end
-    end
-
-    -- find package
-    local result = nil
-    if pkgconfig_file then
+    
+    -- we iterate over each pkgconfig file to extract the required data
+    local result = {
+        includedirs = {},
+        linkdirs = {},
+        links = {},
+        version = nil
+    }
```

Comment:
and here

Reply:
I added `return nil` when the list, cygpath or the version are not found.

Reply:
please use single line and remove `version = nil`

```lua
    local result = {includedirs = {}, linkdirs = {}, links = {}}
```